docs/source/dev-guide/plugins/index.rst

```diff
@@ -56,67 +52,77 @@ a package out of the custom plugin hook shown above:
    build-backend = "setuptools.build_meta"
 
    [project]
-   name = "my-conda-plugin"
+   name = "example"
    version = "1.0.0"
-   description = "My conda plugin"
-   requires-python = ">=3.7"
+   description = "Example conda plugin"
+   requires-python = ">=3.8"
    dependencies = ["conda"]
 
    [project.entry-points."conda"]
-   my-conda-plugin = "my_plugin"
-
-
-The ``setup.py`` file below is an alternative to the ``pyproject.toml`` file shown
-above; its main difference is the ``entry_points`` argument that is provided to the
-``setup()`` function:
+   example = "example"
 
 .. code-block:: python
    :caption: setup.py
 
    from setuptools import setup
 
    setup(
-       name="my-conda-plugin",
+       name="example",
        install_requires="conda",
-       entry_points={"conda": ["my-conda-plugin = my_plugin"]},
+       entry_points={"conda": ["example = example"]},
        py_modules=["my_plugin"],
    )
 
+In both examples shown above, we define an entry point for conda. It's important to make sure
+that the entry point is for "conda" and that it points to the correct module in your plugin package.
+Our package only consists a single Python module called ``example``. If you have a large project,
+be sure to always point the entry point to the module containing the plugin hook declarations (i.e.
+where ``conda.plugins.hookimpl`` is used).
 
-A note on licensing
--------------------
-
-When licensing plugins, we recommend using licenses such as BSD-3_, MIT_, and
-`Apache License 2.0`_. Some ``import`` statements may possibly require the GPLv3_
-license, which ensures that the software being licensed is open source.
-
-Ultimately, the authors of the plugins can decide which license is best for their particular
-use case. Be sure to credit the original author of the plugin, and keep in mind that
-licenses can be altered depending on the situation.
-
-For more information on which license to use for your custom plugin, please reference
-the `"Choose an Open Source License"`_ site.
+More examples
+-------------
 
+To see more examples of conda plugins, please visit the following resources:
 
-API reference
--------------
+- `conda-plugins-template`_: this is a repository with full examples that could be used a starting point for your plugin
 
-.. autoattribute:: conda.plugins::hookimpl
-   :no-value:
```

Comment:
Yeah, that's a far ask.

Otherwise, it just looked really weird having this there:

<img width="385" alt="Bildschirmfoto 2023-06-20 um 12 13 44" src="https://github.com/conda/conda/assets/718726/f785e5f5-034e-463d-97a5-e2b5fa7cfc56">

I'm not sure that would really make sense for a new reader.

Reply:
Sure, let's fix the docstring of the hookimpl

Reply:
Even if the doc string was better. I still do not think it would be a good fit for this particular page. The quick start already shows how to use it. I think this would be a better fit for a dedicated "API" page where users could go for a further detailed reference.